### ISSUE TITLE
SMS scroll up

### ISSUE THREAD

**Issue by Thieb67k** [ubports/messaging-app#66]
Description of the feature
SMS conversation scroll up with no demand.

When I open SMS (at least via notifications), the conversation is opened but scroll up without request.

- Device: LGE Nexus 4
- Channel: Stable
- Build: OTA-5

Steps to reproduce
When I open SMS app, at least via notifications.

Expected behavior
I expect to see the very last text, on the bottom of the conversation.

Actual behavior
I have to scroll down before responding.

**Comment by c0degunner** [comment]
I confirm I get this too, an inconvenience but not a major one. On OnePlus one. Ota7

**Comment by abkro** [comment]
This happens on my Nexus 5 also, rc, 2019-W22.

**Comment by cyberpunkedu** [comment]
This is still happening on Nexus 5 dev channel. Happens when I send a message too.

**Comment by Flohack74** [comment]
Yes happens a lot for me, too...

**Comment by dadandalive** [comment]
On Nexus 5 rc OTA-10 this is not (any longer) happening. However, just after sending off a message (i.e. when clicking the send button), the viewing panel scrolls up, so one has to scroll down to see whether the message was correctly sent off. Please advise whether this can be considered the same bug or whether a different bug ticket needs to be opened.

**Comment by Flohack74** [comment]
I think its the sam bug, so we should be fine with that ;)

**Comment by ateliernz** [comment]
happens to me too on FP2 OTA-11

**Comment by lduboeuf** [comment]
Model in threadInformation List https://github.com/ubports/messaging-app/blob/xenial/src/qml/Messages.qml#L1341 is reloaded twice when sending a message, apparently one event when message status = unknown and one when message status =MessageStatusAccepted . Which update participants list and triggers a model update in Message List twice. Sometimes, it works because the two reload event occurs quickly, sometimes not, which let the scroll position in the middle because it moved before the second event.

What could be done is to check whether effectively the participant list has changed before triggering an update in Message List.

**Comment by dbdb78** [comment]
I experience this on my N5 too. Why is the "issue" closed?

**Comment by UniversalSuperBox** [comment]
TL;DR: The issue is fixed, but not in `stable`.

Issues are closed when the fix for them is slated for inclusion in the daily `devel` channel, when they are added to QA. In QA, we confirm that the issues are fixed on `devel`, then on the weekly `rc` image. Once the `rc` image is ready for release, we release an OTA-## update to `stable`. The next update is OTA-12.

We are currently not shipping weekly `rc` images due to a few critical issues: https://github.com/ubports/ubuntu-touch/labels/critical%20%28devel%29. After `rc` is shipped, we can work on shipping OTA-12, which will include this fix.

**Comment by dbdb78** [comment]
@UniversalSuperBox Thanks for the info. Something to look forward to. OTA-12!

**Comment by Danfro** [comment]
Fix confirmed on E5 RC. When opening a sms thread from the list, the view is opened/scrolled at the position of the last message.
